code/FBX/FBXConverter.cpp

```diff
@@ -68,17 +68,22 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <sstream>
 #include <iomanip>
 #include <cstdint>
+#include <iostream>
+#include <string>
+#include <float.h>
+
+#define FBX_ONE_SECOND 46186158000L
 
 
 namespace Assimp {
     namespace FBX {
 
         using namespace Util;
 
-#define MAGIC_NODE_TAG "_$AssimpFbx$"
-
-#define CONVERT_FBX_TIME(time) static_cast<double>(time) / 46186158000L
+        #define MAGIC_NODE_TAG "_$AssimpFbx$"
 
+        #define CONVERT_FBX_TIME_TO_SECONDS(time) static_cast<double>(time) / 46186158000L
```

Comment:
This allows cleaner time conversions in code, and allows you to be more explicit.